interface-elements-for-desktop/articles/rich-text-editor/tables/wrap-text-around-a-table.md

```diff
@@ -0,0 +1,16 @@
+# Wrap Text Around a Table
+
+* Place the cursor within the target table.
+* Click the **Properties** button in the **Table** group on the **Table Tools/Layout** [tab](../text-editor-ui/ribbon-interface.md) or invoke the context menu and select **Table Properties**.
+
+    ![RTETablesPropertiesRibbon](../../../images/img121442.png)
+
+    ![RTETablesPropertiesContextMenu](../../../images/img121443.png)
+* In the invoked **Table Properties** dialog, click **Around** in the **Text Wrapping** section. Click the enabled **Positioning** button to open the **Table Positioning** dialog.
+
+    ![RTETablesPropertiesRibbon](../../../images/richedit-tables-wrapping-dialog.png)
+* Here you can specify the table's horizontal and vertical position. Enter numeric value in the **Position** combo box to specify an absolute position. To set a relative position, select the required alignment from the **Position** list and the relative element from the **Relative to:** list.
+
+* Specify the distance between the table and surrounding text in the **Distance from surrounding text** section.
+
+* Check **Move with Text** to make the table move with surrounding text.
```

Comment:
Can you clarify what you mean by "make the table move with surrounding text" - for my understanding?

Reply:
the table can be fixed - when new text is added or removed around the table, the table stays still. When you check this box, the table will move depending on the added or removed text

Reply:
Okay, thank you.